modules/build-macros/src/main/scala-2.13/scala/build/EitherCps.scala

```diff
@@ -10,7 +10,7 @@ object EitherCps {
   final case class Helper[E]() {
```

Comment:
Maybe we could remove the `case` here? (seems it was superfluous, and that would justify the use of `new` below)